streams/src/main/java/org/apache/kafka/streams/state/internals/VersionedKeyValueToBytesStoreAdapter.java

```diff
@@ -115,7 +115,7 @@ public boolean persistent() {
 
     @Override
     public boolean isOpen() {
-        return inner.persistent();
+        return inner.isOpen();
```

Comment:
Can we extract this and cherry-pick to `3.5` before the release goes out?

Reply:
Good point. Here: https://github.com/apache/kafka/pull/13695